doc/sphinx/language/core/records.rst

```diff
@@ -417,9 +417,9 @@ the :attr:`projections(primitive)` :term:`attribute`  must be supplied.
 
 .. flag:: Printing Primitive Projection Parameters
 
-   This compatibility :term:`flag` reconstructs internally omitted parameters at
-   printing time (even though they are absent in the actual AST manipulated
-   by the kernel).
+   This compatibility :term:`flag` (off by default) reconstructs
+   internally omitted parameters at printing time (even though they
+   are absent in the actual AST manipulated by the kernel).
```

Comment:
```suggestion
   When set, internally omitted parameters are reconstructed at printing time
   even though they are absent in the AST manipulated by the kernel.  Default is unset.
```

Reply:
Not interested in touching this more in this PR